package.json

```diff
@@ -69,6 +69,7 @@
 		"preact-i18nline": "^2.0.0",
 		"preact-router": "^2.5.7",
 		"query-string": "6.2.0",
+		"react-responsive": "^6.1.1",
```

Comment:
We're not using React in this project. `preact-compat` is applied only for compatibility with Storybook.

Reply:
Okay, I'll change some alternative :)

Reply:
hey @tassoevan can I use preact-fluid ?
it also has grid system.

Reply:
let me implement and test it.

Reply:
I don't recommend using preact-fluid because we're doing here from scratch exactly what preact-fluid is: a UI toolkit. Also, I don't think the tooltips on preact-fluid calculate boundaries to reposition themselves, one of requirements of #157.

Reply:
Hey @tassoevan 
Sorry for the late reply I was busy with my Midsem exams,
Ya, actually yesterday I was figuring out the way to do media queries through preact and react but not able to find the way from scratch. can I use CSS plugin for that or please suggest the way :)